packages/payment-backend/src/errors.ts

```diff
@@ -88,6 +88,18 @@ export class PaymentNotFound extends ExperienceError {
   }
 }
 
+export class PaymentsNotFound extends ExperienceError {
+  constructor() {
+    super({
+      code: 'payments-not-found',
+      message: 'Payments not found',
+      logLevel: 'info',
+      responseStatus: StatusCode.NotFound,
+    })
+    Object.setPrototypeOf(this, PaymentNotFound.prototype)
```

Comment:
Should probably be PaymentsNotFound.prototype (instead of PaymentNotFound)

Reply:
Yeh, I will fix this.